compiler/daml-lf-proto-encode/src/DA/Daml/LF/Proto3/EncodeV2.hs

```diff
@@ -935,10 +913,9 @@ encodeDefDataType DefDataType{..} = do
             defDataType_FieldsFields <- encodeFieldsWithTypes unVariantConName fs
             pure $ P.DefDataTypeDataConsVariant P.DefDataType_Fields{..}
         DataEnum cs -> do
-            mangledAndInterned <- encodeNames (map unVariantConName cs)
-            let (defDataType_EnumConstructorsConstructorsStr, defDataType_EnumConstructorsConstructorsInternedStr) = case mangledAndInterned of
-                    Left mangled -> (V.fromList mangled, V.empty)
-                    Right mangledIds -> (V.empty, V.fromList mangledIds)
+            mangledIds <- encodeNames (map unVariantConName cs)
+            let defDataType_EnumConstructorsConstructorsStr = V.empty
```

Comment:
May be worth making a TODO here for eventually removing the constructors_str field from proto, like e.g. on 115 of compiler/daml-lf-proto-encode/src/DA/Daml/LF/Proto3/EncodeV2.hs above.

Reply:
Done